spec.md

```diff
@@ -163,9 +163,15 @@ Service Broker using HTTP basic authentication (the `Authorization:` header)
 on every request. This specification does not specify how Platform and Service
 Brokers agree on other methods of authentication.
 
+Platforms and Service Brokers MAY agree on an authentication mechanism other
+than basic authentication, but the specific agreements are not covered by this
+specification. Please see the [profile](profile.md) documentation for a
+description of each mechanism and platform support.
+
 If authentication is used, the Service Broker MUST authenticate the request
-using the predetermined authentication mechanism and MUST return a `401
-Unauthorized` response if the authentication fails.
+using the predetermined authentication mechanism, securing communications
+via TLS, and MUST return a `401 Unauthorized` response if the authentication
```

Comment:
A dependency on TLS makes service broker development more difficult. For production TLS is a must, for development not necessarily. If we insist on TLS, do we allow self-signed certificates?

Reply:
I'd be ok with self-signed certs for non-production deployments, or honestly even not requiring TLS in those cases

Reply:
Per offline conversation with @duglin, @jboyd01 and myself, we will change this language to require TLS only in production, and say that it's a decision between the broker and the platform whether they use root or self signed certs